source/models/defining-models.md

```diff
@@ -180,6 +174,18 @@ export default DS.Model.extend({
 });
 ```
 
+#### Synchronous relationships
+
+Starting with Ember Data 2.0 all relationships are asynchronous by default. If
```

Comment:
"asynchronous" isn't defined. Maybe something like "Calling a relationship, like `myPost.get('comments')`, returns a promise. If you want the value to be returned instead, you need to pass an `async` option set to `false`:".

Or maybe we should just omit, this, since isn't it generally a bad idea and not something most people would want to do?